test-data/local/poly_example/poly.ert

```diff
@@ -16,3 +16,6 @@ GEN_DATA POLY_RES RESULT_FILE:poly_%d.out REPORT_STEPS:0 INPUT_FORMAT:ASCII
 
 INSTALL_JOB poly_eval POLY_EVAL
 SIMULATION_JOB poly_eval
+-- Load the RML module, this does not imply that it is actually used. That will
```

Comment:
Could you please move this to a separate config file (`poly_rml_enkf.ert` or similar)? I really like the simplicity of the `poly.ert` config file and I'm planning to use it in early training of new developers...

Reply:
Done